crates/holochain/src/core/ribosome/guest_callback/validate.rs

```diff
@@ -307,55 +307,6 @@ mod slow_tests {
         assert_eq!(result, ValidateResult::Valid,);
     }
 
-    #[tokio::test(flavor = "multi_thread")]
-    async fn test_validate_implemented_invalid_return() {
```

Comment:
The test wasms no longer compile with invalid return types, therefore these tests are not required.

Reply:
As above, these are testing two different things. `hdk_derive` is checking the provided code and helping you build a WASM but for Holochain, a WASM could have come from anywhere, like an old version of `hdk_derive` without the new code checks. So it's still important that Holochain doesn't crash.

Reply:
The tests that depend on non-compiling `TestWasm` tests have now been modified to use inline zomes